app/models/patient.rb

```diff
@@ -88,6 +89,7 @@ class Patient
   field :patient_contribution, type: Integer
   field :naf_pledge, type: Integer
   field :fund_pledge, type: Integer
+  field :fund_pledged_at, type: Time
```

Comment:
Should this field follow onto archived patients?

Reply:
Not sure! My inclination is no because it's more of an app logic oriented field that we kind of obscure from people (a  la created_at/updated_at) but @mdworken @alisonnjones  let me know if we should take a diff tack.